spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -529,6 +531,30 @@ private void publishConsumerFailedToStart() {
 		}
 	}
 
+	private void publishRetryAuthEvent(Throwable throwable) {
```

Comment:
Please, add your name to the `@author` list: you have made some changes to the class, so you deserve some credit.